templates2/display/silo.html

```diff
@@ -208,16 +212,33 @@ <h4 class="modal-title" id="popuptitle"></h4>
         <div class="panel-heading">
             <h3 class="panel-title"><a href="/silo_edit/{{ silo.id }}">{{silo.name}}</a></h3>
             <small>{{ silo.description|default:"" }}</small>
+            {% if tasks_running %}
+                <h4>Background processes runnning</h4>
+                <small style="color:#ff3019">
+                    Data imports are still running in the background.
+                    Silo is unavailable until all processes are finished. # of processes: {{ tasks_running }}
```

Comment:
I don't think the users need to know all these things, so the message can be shorter. `The Table is not available because the importing process did not finish yet. `